dotnet/test/support/PageObjects/ByAllTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Collections.Generic;
```

Comment:
@lukeis ok, will add, I aligned that one with the existing tests on ByChainedTests which also does not have a copyright notice. Should I align those files as well?